apollo-router/src/query_planner/execution.rs

```diff
@@ -3,7 +3,8 @@ use std::sync::Arc;
 
 use futures::future::join_all;
 use futures::prelude::*;
-use tokio::sync::broadcast::Sender;
+use tokio::sync::broadcast;
```

Comment:
why the broadcast? The defer implementation only needed an mpsc before. Andthe broadcast channel has a very different performance profile

Reply:
That was already there. I've just preserved the existing behaviour. If I haven't, then it's a mistake, but I just checked and all my changes look consistent to me.